packages/block-library/src/post-time-to-read/edit.js

```diff
@@ -93,6 +126,45 @@ function PostTimeToReadEdit( { attributes, setAttributes, context } ) {
 					} }
 				/>
 			</BlockControls>
+			<InspectorControls>
+				<ToolsPanel
+					label={ _x( 'Time to Read', 'Panel title' ) }
```

Comment:
```suggestion
					label={ __( 'Settings' ) }
```

It is better to unify the `ToolsPanel` label with Settings, just like other blocks.

Reply:
Makes sense! Done in https://github.com/WordPress/gutenberg/pull/71606/commits/b0146755dc4921dfd45a2dfe7395873d405cd904.